content/doc/upgrade-guide/2.150.adoc

```diff
@@ -0,0 +1,17 @@
+---
+layout: documentation
+title:  Jenkins Upgrade Guide
+notitle: true
+---
+
+== Upgrading to Jenkins LTS 2.150.x
+
+Each section covers the upgrade from the previous LTS release, the section on 2.138.1 covers the upgrade from 2.138.4.
+
+=== Upgrading to Jenkins LTS 2.150.1
+
+NOTE: 2.150.1 and 2.138.4 were released on the same day.
+Please note that this upgrade guide only covers the upgrade from 2.138.4.
+See link:../2.138/[its upgrade guide] for information related to upgrading from 2.138.3.
+
+No notable changes requiring upgrade notes.
```

Comment:
I don't think anything in 2.150.1 is notable enough here, but I may be wrong about some of the dependency updates towards Java 11.